src/System.ServiceModel.Syndication/src/System/ServiceModel/Syndication/TextSyndicationContent.cs

```diff
@@ -24,29 +17,22 @@ public TextSyndicationContent(string text) : this(text, TextSyndicationContentKi
 
         public TextSyndicationContent(string text, TextSyndicationContentKind textKind)
         {
-            if (!TextSyndicationContentKindHelper.IsDefined(textKind))
+            if (!Enum.IsDefined(typeof(TextSyndicationContentKind), textKind))
             {
```

Comment:
Enum.IsDefined is very very (unbelievable so) slow, that's why the helper exists. Please revert the change back to using TextSyndicationContentKindHelper

Reply:
Nice alternative!

---
In reply to: [192255407](https://github.com/dotnet/corefx/pull/29882#discussion_r192255407) [](ancestors = 192255407)